NitroxServer-Subnautica/Serialization/Resources/Parsers/Monobehaviours/PrefabIdentifierParser.cs

```diff
@@ -10,12 +10,12 @@ public class PrefabIdentifierParser : MonobehaviourParser
 
         public static Dictionary<string, AssetIdentifier> GameObjectIdByClassId { get; } = new Dictionary<string, AssetIdentifier>();
 
-        public override void Parse(AssetIdentifier identifier, AssetIdentifier gameObjectIdentifier, AssetsFileReader reader, ResourceAssets resourceAssets)
+        public override void Parse(AssetIdentifier identifier, AssetIdentifier gameObjectIdentifier, AssetsFileReader reader, ResourceAssets resourceAssets, Dictionary<int, string> relativeFileIdToPath)
         {
             string classId = reader.ReadCountStringInt32();
 
-            ClassIdByGameObjectId.Add(gameObjectIdentifier, classId);
-            GameObjectIdByClassId.Add(classId, gameObjectIdentifier);
+            ClassIdByGameObjectId[gameObjectIdentifier] = classId;
+            GameObjectIdByClassId[classId] = gameObjectIdentifier;
```

Comment:
Let's revert back so we get warned on dupes.  Same with all of the other files below.